pkgs/applications/version-management/rcshist/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation {
+  pname = "rcshist";
+  version = "1.04";
+
+  src = fetchurl {
+    url = "https://invisible-island.net/datafiles/release/rcshist.tar.gz";
```

Comment:
```suggestion
    url = "https://web.archive.org/web/20220508220019/https://invisible-island.net/datafiles/release/rcshist.tar.gz";
```

Reply:
Why? We normally use upstream servers directly, afaik.

Reply:
Because that archive has no version and is not reproducible. 

@happysalada This is not a stylistic change which can be simple ignored.

Reply:
My bad for missing this!

@KAction how do you feel about making the change ? If for any reason you can't, no worries, just let me know and I will do it myself.

Reply:
Sure, https://github.com/NixOS/nixpkgs/pull/172339

Reply:
Thank you!

Reply:
also, if you have a PR that fails to get attention, feel free to ping me.
I'm happy to help until you manage to get those commiters rights.
Thank you for your contributions and welcome to the community!